integration-tests/test_connection.py

```diff
@@ -0,0 +1,105 @@
+import pytest
+import conftest
+
+
+pytestmark = pytest.mark.usefixtures("register_subman")
+
+
+def test_connection(insights_client):
+    """
+    Test insights-client --test-connection
+    `test_connection` method returns 0 if success, 1 if failure
+    """
+    url_test = "End Upload URL Connection Test: SUCCESS"
+    api_test = "End API URL Connection Test: SUCCESS"
+
+    assert conftest.loop_until(lambda: insights_client.is_registered)
```

Comment:
this will actually fail, as each test starts with an unregistered `insights-client`; simply drop this